### PR TITLE
remove duplicated vsphere-8 ci jobs

### PR DESCRIPTION
**Pull request by jianlinliu**
`vsphere-8` ci jobs were added to cover vsphere 8 platform when our main vsphere test farm is not up-to-date yet, now our main vsphere test farm is already updated to 8, so remove the duplicated ones.

Also move a vsphere disasterrecovery job from the temp vsphere-8 platform to our main test farm, also update its job name (vsphere never had a private cluster concept)

### CODE REVIEW

**Comment by liangxia** [comment]
@jianlinliu Looks like you forgot to run `make update` to update the jobs ?

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.16__amd64-nightly.yaml [hunk 1]
```diff
@@ -651,13 +651,17 @@ tests:
...
     - ref: openshift-extended-test-longduration
     workflow: cucushift-installer-rehearse-aws-ipi-private
-- as: vsphere-8-ipi-private-proxy-f14-disasterrecovery
+- as: vsphere-ipi-proxy-f14-disasterrecovery
   cron: 11 1 8,22 * *
   steps:
-    cluster_profile: vsphere-8-vpn
+    cluster_profile: vsphere-dis-2
     env:
+      FIPS_ENABLED: "true"
```
Comment (liangxia) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.16__amd64-nightly.yaml:659]: The test name does not indicate fips. Perhaps update the test name or remove this line to make them in sync for fips status.

Comment (jianlinliu) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.16__amd64-nightly.yaml]: updated.

**Pull request merged**